<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LemeFind</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>421</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Rules</string>
    </property>
    <widget class="QCheckBox" name="ignoreCmtCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ignore comment</string>
     </property>
    </widget>
    <widget class="QComboBox" name="operatorComboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="number1TextBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="numOfResultsCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>53</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of results</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>332</x>
       <y>52</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>and</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="number2TextBox">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="caseSensitiveCheckBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Case sensitive</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="wholeWordCheckBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Whole word</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>421</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Filters</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Files to include:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="fileToIncludeComboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>311</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QComboBox" name="fileToExcludeComboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>311</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Files to exclude:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>( * = any string; , = seperate)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <widget class="QPushButton" name="searchInputBrowse">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>20</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Search input:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Search directory:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="searchInput">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>281</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QComboBox" name="searchDirectory">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>281</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="searchDirectoryBrowse">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>60</y>
       <width>20</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>421</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPlainTextEdit" name="consoleBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>301</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLabel" name="fileNameLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="outputButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>91</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>443</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionReadme"/>
    <addaction name="separator"/>
    <addaction name="actionUpdate"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionReadme">
   <property name="text">
    <string>Readme</string>
   </property>
  </action>
  <action name="action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionCheck_for_updates_2">
   <property name="text">
    <string>Check for updates</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Check for updates</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About LemeFind</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
